pygen/pygen_src/isa/riscv_instr_cov.py

```diff
@@ -5,27 +5,27 @@
 from pygen_src.riscv_instr_pkg import riscv_reg_t
 
 class operand_sign_e(Enum):
-    POSITIVE = auto()
+    POSITIVE = 0
```

Comment:
why hardcode these to zero?

Reply:
No specific reason actually, just wanted to have an auto enum starting from 0.

Reply:
ah, I just checked the auto() documentation and seems like it starts from 1 by default - I'm fine starting these from 0.